src/NumericalAlgorithms/LinearOperators/ApplyMatrices.hpp

```diff
@@ -129,4 +129,4 @@ ResultType apply_matrices(const std::array<MatrixType, Dim>& matrices,
   apply_matrices(make_not_null(&result), matrices, u, extents);
   return result;
 }
-//@}
+// @}
```

Comment:
Move this to a different commit or drop it.  (It is the only change to this file in this commit.)